packages/databricks-vscode/src/language/ConfigureAutocomplete.ts

```diff
@@ -43,7 +44,7 @@ export class ConfigureAutocomplete implements Disposable {
     constructor(
         private readonly context: ExtensionContext,
         private readonly stateStorage: StateStorage,
-        private readonly workspaceFolder: string,
+        private readonly workspaceFolderManager: WorkspaceFolderManager,
```

Comment:
Should we listen to changes here an re-run `configure`?

Reply:
Configure autocomplete has always been run on demand (by running the command). I do not want to change that behaviour in this PR.

Reply:
Hmm, that `configure` call in the constructor is confusing, but I see that it doesn't do anything